src/lib/Components/ArtworkGrids/Artwork.tsx

```diff
@@ -21,6 +22,26 @@ interface Props {
   onPress?: (artworkID: string) => void
 }
 
+const Badges = styled.View`
```

Comment:
All the style changes are me just trying to overlay the badge in the lower left of the image, some styling for the badge itself, etc. Probably can/should be cleaned up once I get it figured out. What we do on web doesn't seem to directly carry over: https://github.com/artsy/reaction/blob/71a9e803bf235946ad3992694fce97a151a9d55a/src/Components/Artwork/GridItem.tsx#L43-L53